packages/gatsby-theme-newrelic/src/components/SearchInput.js

```diff
@@ -97,18 +98,18 @@ const SearchInput = forwardRef(
           }}
           css={css`
             width: 100%;
-            border: 1px solid var(--border-color);
+            border: 0px solid var(--border-color);
             border-radius: 4px;
-            background: var(--primary-background-color);
+            background: var(--color-neutrals-700);
             color: var(--primary-text-color);
             transition: 0.15s ease-out;
-            line-height: 1;
+            line-height: 1.5rem;
 
             padding-left: calc(
-              var(--horizontal-spacing) + 0.5rem + var(--icon-size)
+              var(--horizontal-spacing) + 0.825rem + var(--icon-size)
             );
             padding-right: calc(
-              var(--horizontal-spacing) + 0.5rem + var(--icon-size)
+              var(--horizontal-spacing) + 0.825rem + var(--icon-size)
```

Comment:
These changes will affect all search inputs. Just to be sure, are you meaning to update this for all search inputs or just the search input in the global header?